postman/Login_test

```diff
@@ -0,0 +1,195 @@
+{
+	"info": {
+		"_postman_id": "cdaee4f6-9167-4df8-8e54-3f10e5dcdab5",
+		"name": "Login_Form",
+		"schema": "https://schema.getpostman.com/json/collection/v2.1.0/collection.json"
+	},
+	"item": [
+		{
+			"name": "Post",
+			"item": [
+				{
+					"name": "API test",
+					"event": [
+						{
+							"listen": "prerequest",
+							"script": {
+								"exec": [
+									""
+								],
+								"type": "text/javascript"
+							}
+						},
+						{
+							"listen": "test",
+							"script": {
+								"exec": [
+									"var jsonData = JSON.parse(responseBody);\r",
+									"postman.setEnvironmentVariable(\"token\", jsonData.token);\r",
+									"\r",
+									"pm.test(\"Status code is 200\", function () {\r",
+									"    pm.response.to.have.status(200);\r",
+									"});\r",
+									"\r",
+									"pm.test(\"body_test\", function () {\r",
+									"    pm.expect(pm.response.code).to.be.oneOf([200, 202]);\r",
+									"});\r",
+									"\r",
+									"pm.test(\"Contains a message property\", function() {\r",
+									"    let jsonData = pm.response.json();\r",
+									"    \r",
+									"});\r",
+									"\r",
+									"/*signup*/\r",
+									"pm.test(\"Response has a JSON body\", function () {\r",
+									"    pm.response.to.be.json;\r",
+									"});\r",
+									"\r",
+									"pm.test(\"Response has a token property\", function () {\r",
+									"    var jsonData = pm.response.json();\r",
+									"    pm.expect(jsonData.token).to.be.a('string');\r",
+									"});\r",
+									"\r",
+									"/*signin*/\r",
+									"pm.test(\"Response has a data property\", function () {\r",
+									"    var jsonData = pm.response.json();\r",
+									"    pm.expect(jsonData).to.be.a('object');\r",
+									"});\r",
+									"\r",
+									"\r",
+									"pm.test(\"400 invalid request\", function () {\r",
```

Comment:
this is failing

Reply:
I see you have provided a different image, those tests are not provided in the PR!
Moreover, the coverage is not what we were looking forward to